src/devices/Dhtxx/DhtBase.cs

```diff
@@ -128,7 +115,7 @@ private void ReadData()
         /// <returns>
         /// <c>true</c> if read is successfull, otherwise <c>false</c>.
         /// </returns>
-        private bool ReadThroughOneWire()
+        internal virtual void ReadThroughOneWire()
```

Comment:
Wasn't the whole point of IsLastReadSuccessful to make sure that we got a valid accurate reading? if you are removing it, how will we be certain of that now? Also, the xml docs above the method will have to be reflected with this change.

Reply:
Because DHT10 does not provide validation, I just let this method not return bool values. `IsLastReadSuccessful` still gives value.